node/src/accountant/payment_adjuster/mod.rs

```diff
@@ -442,18 +437,18 @@ impl AdjustmentAnalysis {
 
 #[derive(Debug, PartialEq, Eq, VariantCount)]
 pub enum PaymentAdjusterError {
-    EarlyNotEnoughFeeForSingleTransaction {
+    AbsolutelyInsufficientBalance {
         number_of_accounts: usize,
         transaction_fee_opt: Option<TransactionFeeImmoderateInsufficiency>,
         service_fee_opt: Option<ServiceFeeImmoderateInsufficiency>,
     },
-    LateNotEnoughFeeForSingleTransaction {
+    AbsolutelyInsufficientServiceFeeBalancePostTxFeeAdjustment {
```

Comment:
I think we can plan to shorten the name by using new field(s). I hope that's the case.